src/test/suite/completion.task.test.ts

```diff
@@ -3,14 +3,14 @@ import * as vscode from 'vscode';
 import { getDocUri } from './util';
 
 suite('Should do completion in tasks.json', () => {
-	
-	
+
```

Comment:
why two empty lines? one should be enough 🙂

Reply:
Lol, there were two lines prettier just delete tab xD, will fix it too

Reply:
BTW, I would like to delete both lines, I think there is no need in one also

Reply:
BTW, I would like to delete both lines, I think there is no need in one also

Reply:
it depends, but ok